ui/components/app/transaction-breakdown/transaction-breakdown.component.js

```diff
@@ -182,11 +196,30 @@ export default class TransactionBreakdown extends PureComponent {
             )}
           </TransactionBreakdownRow>
         )}
+        {isOptimism && (
```

Comment:
I see we are simply checking if we are on the optimism network in order to display this extra info, will this present any issue for transactions that were previously on optimism, but before the hard fork?